package.json

```
@@ -59,11 +59,16 @@
     "@types/react-router-dom": "^5.1.7",
     "@types/webpack": "^5.28.0",
     "@types/webpack-dev-server": "^3.1.6",
+    "@typescript-eslint/eslint-plugin": "^4.28.1",
+    "@typescript-eslint/parser": "^4.28.1",
     "babel-loader": "^8.2.2",
     "clean-webpack-plugin": "^3.0.0",
     "copy-webpack-plugin": "^6.4.1",
     "css-loader": "^5.2.4",
     "eslint": "^7.29.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-office-addins": "^0.2.0",
     "eslint-plugin-react": "^7.24.0",
```

Oups, I think the instructions were missing the `react-hooks` dependency ([they suggest](https://www.npmjs.com/package/eslint-config-airbnb) a lower version number but 4.2 seems to work fine). Sorry about that, I have no idea why it didn't generate an error.

```suggestion
    "eslint-plugin-react": "^7.24.0",
    "eslint-plugin-react-hooks": "^4.2.0",
```